apps/i18n/common/es_es.json

```diff
@@ -1236,6 +1255,7 @@
   "printReportWindowTitle": "Informe de Normas para {sectionName}",
   "privacyDocExplanation": "Proteger la privacidad de los estudiantes es muy importante para nosotros. Te recomendamos que compartas esta carta con tus padres para que conozcan Code.org y puedan revisar nuestras políticas de privacidad.",
   "privacyHeading": "Carta a padres y privacidad de los estudiantes",
+  "privacyLinkToPolicy": "¿Solo busca una carta sobre la política de privacidad de Code.org para familias? [Marque aquí] ({privacyPolicyLink}).",
```

Comment:
FYI @castro-jorge that it looks like a bunch of these Spanish translations broke some markdown